nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexSurefireExecution.java

```diff
@@ -91,6 +91,34 @@ protected void setupArgline() {
 
     }
 
+    @Override
+    protected Xpp3Dom applyNonDexConfig(Xpp3Dom configuration) {
+        return addExcludedGroups(super.applyNonDexConfig(configuration));
+    }
+
+    private Xpp3Dom addExcludedGroups(Xpp3Dom configNode) {
```

Comment:
Why are we moving this to NonDex Surefire Execution? I thought we should apply the ignore everywhere.
This moving didn't fix the issue - it just changed so that tests were now not run with nondex while they were run in clean. Are you sure this needs to be moved?